packages/dm-core-plugins/blueprints/table/TablePluginConfig.json

```diff
@@ -20,6 +20,11 @@
       "default": [],
       "dimensions": "*"
     },
+    {
+      "attributeType": "CORE:ViewConfig",
+      "type": "CORE:BlueprintAttribute",
+      "name": "expandableRecipeViewConfig"
+    },
```

Comment:
Note to self: Should also add one for openAsTab at some point

Reply:
But isn't openAsTab just opening the object and should use the default view? I asked Babak and he said yes when I asked if it should just be the ui recipe for the object

Reply:
Mh, okay. Eirik Ola thought this should be configurable. But we can discuss this on the Monday meeting. Either way, it does not need to be fixed right away